_posts/2018-06-25-java-releases.md

```diff
@@ -11,234 +11,237 @@ headerImage: post
+machines contain. If you work with a cloud based infrastructure things will become much harder to understand because the real
 CPUs are shared between multiple virtual machines or containers (like in Docker and Kubernetes).
 
 Next to this model that is usable for Java on the server Oracle provides a special support for Java on the desktop. Here the
-price for the commercial support is based on the number of users / client terminals. The following table shows the cost for 
+price for the commercial support is based on the number of users / client terminals. The following table shows the cost for
 the "Java SE Desktop Subscription" model:
 
 {:.table}
 | Users / client machines | Monthly price per user / client machine |
 | ----------------------- | --------------------------------------- |
-| 1-999                   | $2.50                                   | 
-| 1,000-2,999             | $2.00                                   | 
-| 3,000-9,999             | $1.75                                   | 
-| 10,000-19,999           | $1.50                                   | 
-| 20,000-49,999           | $1.25                                   | 
-| 50,000+                 | Oracle must be contacted for details    | 
-
-Commercial support for Java applications on the desktop might become quite important for some companies since Oracle will drop 
-several important desktop features from the JDK starting with Java version 11. If you are using Java on the desktop I highly 
+| 1-999                   | $2.50                                   |
+| 1,000-2,999             | $2.00                                   |
+| 3,000-9,999             | $1.75                                   |
+| 10,000-19,999           | $1.50                                   |
+| 20,000-49,999           | $1.25                                   |
+| 50,000+                 | Oracle must be contacted for details    |
+
+Commercial support for Java applications on the desktop might become quite important for some companies since Oracle will drop
+several important desktop features from the JDK starting with Java version 11. If you are using Java on the desktop I highly
 recommend to read [this article about the Java client roadmap that was announced by Oracle in 2018](https://dzone.com/articles/what-the-future-java-releases-will-mean-for-legacy).
 
-If you need more information about the commercial support for Java that is offered by Oracle you should have a look at 
+If you need more information about the commercial support for Java that is offered by Oracle you should have a look at
 [this official document](http://www.oracle.com/technetwork/java/javaseproducts/overview/javasesubscriptionfaq-4891443.html). If you
-use Java WebStart technology to provide desktop clients you should take care about the current situation as fast as possible 
+are using Java WebStart technology to provision desktop clients you should take care about the current situation as fast as possible
```

Comment:
is "provision" really right?

Reply:
Honestly I am not 100% sure :laughing: 

Maybe distribute would be better

Reply:
yes, pls use distribute :)

Reply:
done